src/main/kotlin/me/serce/solidity/lang/completion/engine.kt

```diff
@@ -94,3 +102,19 @@ object SolCompleter {
     }.toTypedArray()
   }
 }
+
+class ContractLookupElement(val contract: SolContractDefinition) : LookupElement() {
+  override fun getLookupString(): String = contract.name!!
+
+  override fun renderElement(presentation: LookupElementPresentation) {
+    presentation.icon = SolidityIcons.CONTRACT
+    presentation.itemText = contract.name
+    presentation.typeText = "from ${contract.containingFile.name}"
+  }
+
+  override fun handleInsert(context: InsertionContext?) {
```

Comment:
`context` is never null, so changing this to `(context: InsertionContext)` will help to avoid `!!` on the next line

Reply:
Sure, added